sandpack-client/src/clients/static/index.ts

```diff
@@ -81,17 +81,22 @@ export class SandpackStatic extends SandpackClient {
     }
   }
 
-  private injectDocType(content: FileContent): FileContent {
+  private validateHtml(content: FileContent): FileContent {
     // Make it a string
     let contentString = readBuffer(content);
 
-    // Add the DOCTYPE tag
-    const docTypeRegex = /<!DOCTYPE*>/gi;
-    if (!docTypeRegex.test(contentString)) {
-      contentString = `<!DOCTYPE html>\n${content}`;
-    }
+    // Check how long DOMParser takes
+    console.time('domparser');
+
+    const domParser = new DOMParser();
+    const doc = domParser.parseFromString(contentString, "text/html");
+    doc.documentElement.setAttribute("lang", "en");
+
+    const html = doc.documentElement.outerHTML;
 
-    return contentString;
+    console.timeEnd('domparser');
```

Comment:
I'll obviously remove the `console.time` stuff, but I left it in for now so y'all could see how I was diagnosing the times I mentioned in the PR :)